dd-sdk-android-core/src/main/kotlin/com/datadog/android/api/feature/FeatureScope.kt

```diff
@@ -9,12 +9,18 @@ package com.datadog.android.api.feature
 import androidx.annotation.AnyThread
 import com.datadog.android.api.context.DatadogContext
 import com.datadog.android.api.storage.EventBatchWriter
+import com.datadog.android.api.storage.datastore.DataStoreHandler
 
 /**
  * Represents a Datadog feature.
  */
 interface FeatureScope {
 
+    /**
+     * Property to enable interaction with the DataStore.
```

Comment:
nit: there is no `DataStore` class/reference per se. We need maybe to use `DataStoreHandler` reference or write `data store`

```suggestion
     * Property to enable interaction with the data store.
```